snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -33,7 +33,9 @@
 
 class CommittableTick(NamedTuple):
     tick: Tick
-    should_commit: bool
+    # Offset that we can safely committed once the tick is processed.
+    # Not necessarily the same as the tick's offset.
+    offset_to_commit: Optional[int]
```

Comment:
Could you please adjust the docstring of `ProvideCommitStrategy` (specifically the last paragraph)? It still reference `should_commit`